blocks/init/src/Blocks/components/heading/heading.php

```diff
@@ -37,10 +37,9 @@
 $headingLevel = $headingLevel ? "h{$headingLevel}" : 'h2';
 
 $unique = Components::getUnique();
+echo Components::outputCssVariables($attributes, $manifest, $unique, $globalManifest); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
Should this be escaped? Or can the `phpcs.xml.dist` file be updated to include this method?

Reply:
I think that `phpcs:ignore` is an old leftover from somewhere:
1. We're not using `WordPress.Security` anymore, we're now at `Eightshift.Security`
2. The removed code below doesn't have the ignore 😄